src/xcode/ENA/ENA/Source/Services/HealthCertificate/__test__/HealthCertificateServiceTests.swift

```diff
@@ -125,7 +127,8 @@ class HealthCertificateServiceTests: CWATestCase {
 		// GIVEN
 		let service = HealthCertificateService(
 			store: MockTestStore(),
-			signatureVerifying: DCCSignatureVerifyingStub(error: .HC_COSE_NO_SIGN1),
+			signatureVerifying: DCCSignatureVerifyingStub(),
```

Comment:
why is the test still succeeding when you do not pass an error into it? Is there a logical mistake in the test?

Reply:
It wasn't succeeding anymore, I accidentally deleted the `error:` parameter when adding the list provider mock. Added it back again now :)